src/test/resources/org/jabref/logic/importer/fetcher/collection_of_computer_science_bibliographies_multiple_results_first_result.bib

```diff
@@ -0,0 +1,5 @@
+@misc{,
```

Comment:
The fetcher should generate a BibTeX key. (I currently do not know how this works. Maybe the code of other fetchers can help? Think, its nested in doPostCleanup somehwere)